### PR TITLE
feat(design-system): Icons

### PR DESCRIPTION
**Pull request by th0rOdinson**

### CODE REVIEW

File: apps/root/src/pages/dca/positions/components/positions-list/position-card/index.tsx [hunk 1]
```diff
@@ -334,7 +334,7 @@ export const OpenPosition = ({
           <ContainerBox gap={6} flexDirection="column">
             <StyledCardHeader>
               <ContainerBox gap={2}>
-                <ComposedTokenIcon tokens={[from, to]} size={8} />
+                <ComposedTokenIcon tokens={[from, to]} size={8} overlapRatio={0.6} marginRight={5} />
```
Comment (FiboApe) [inline apps/root/src/pages/dca/positions/components/positions-list/position-card/index.tsx:337]: is this overlapRation a patter that all should follow?

File: apps/root/src/pages/position-detail/components/position-summary-controls/index.tsx [hunk 2]
```diff
@@ -58,6 +59,18 @@ const StyledDivider = styled(DividerBorder1).attrs({ orientation: 'vertical', fl
   margin: ${({ theme }) => theme.spacing(1)} 0;
 `;
 
+const StyledIconButton = styled(IconButton)`
```
Comment (FiboApe) [inline apps/root/src/pages/position-detail/components/position-summary-controls/index.tsx:62]: Seems like we are repeating this one, we should probably extract it away

File: apps/root/src/pages/transfer/components/recipient-address/components/contacts-button/index.tsx [hunk 3]
```diff
@@ -28,8 +27,8 @@ const StyledContactsButton = styled(ForegroundPaper).attrs({ elevation: 0 })`
 
 const ContactsButton = ({ onClick }: ContactsButtonProps) => (
   <StyledContactsButton onClick={onClick}>
-    <Profile2UsersIcon />
-    <Typography variant="bodyExtraExtraSmallBold" color={({ palette }) => colors[palette.mode].accentPrimary}>
+    <Profile2UsersIcon sx={({ palette }) => ({ color: colors[palette.mode].accentPrimary })} />
+    <Typography variant="bodyExtraExtraSmallBold" color="primary">
```
Comment (FiboApe) [inline apps/root/src/pages/transfer/components/recipient-address/components/contacts-button/index.tsx:31]: hold up there cowboy, lets not trust the color property of the typography component

File: packages/ui-library/src/theme/variants/button-variants.ts [hunk 4]
```diff
@@ -17,6 +17,9 @@ export const buildButtonVariant = (mode: 'light' | 'dark'): Components => ({
       },
       endIcon: {
         marginLeft: '2px',
+        '& > *:first-child': {
+          fontSize: '1.25rem', // 20/16
```
Comment (FiboApe) [inline packages/ui-library/src/theme/variants/button-variants.ts:21]: what about line height and other props? its gonna affect it

File: packages/ui-library/src/theme/variants/svgicon-variants.ts [hunk 5]
```diff
@@ -18,16 +18,16 @@ export const buildSvgIconVariant = (mode: 'light' | 'dark'): Components => ({
...
-        },
-      },
-      fontSizeLarge: {
-        fontSize: '2rem', // 32/16
       },
       fontSizeSmall: {
         fontSize: '1.125rem', // 18/16
       },
+      fontSizeMedium: {
+        fontSize: '1.25rem', // 20/16
```
Comment (FiboApe) [inline packages/ui-library/src/theme/variants/svgicon-variants.ts:26]: ah we got the sizes here, perfect

File: apps/root/src/common/components/timeline-controls/dca-items/index.tsx [hunk 6]
```diff
@@ -60,7 +60,7 @@ const buildDcaTimelineTransactionData = (action: DCAPositionAction, chainId: num
       )}
       <Typography variant="bodyRegular">
         <StyledTimelineLink href={buildEtherscanTransaction(action.tx.hash, chainId)} target="_blank" rel="noreferrer">
-          <OpenInNewIcon fontSize="inherit" />
+          <ExportIcon fontSize="small" />
```
[the remark this answers is not in the document]
Comment (th0rOdinson) [reply apps/root/src/common/components/timeline-controls/dca-items/index.tsx:63]: Defined in the theme :D

File: packages/ui-library/src/theme/variants/button-variants.ts (hunk 4 above)
Comment (th0rOdinson) [reply packages/ui-library/src/theme/variants/button-variants.ts:21]: This rule just overrides one heavy selector from MUI on endIcons for buttons. By the way, I was able to fix a related issue: In Figma, icons inside buttons are sort of 'absolute' positioned, so the padding starts when the text ends. This was affecting most of our buttons with icons in the app :D 

Screenshot 2024-10-14 at 5 44 03 PM

**Review by FiboApe — APPROVED** [review]

**Pull request merged**